test/functional/test-cid.js

```diff
@@ -44,6 +44,8 @@ import {stubServiceForDoc} from '../../testing/test-helper';
 
 const DAY = 24 * 3600 * 1000;
 
+const API_KEY = 'AIzaSyA65lEHUEizIsNtlbNo-l2K18dT680nsaM';
```

Comment:
Let's remove this and inline it in the original `isScopedOptedIn_` tests you moved.